tensorflow/training/docker/2.11/py3/cu112/Dockerfile.gpu

```diff
@@ -327,104 +327,105 @@ CMD ["/bin/bash"]
-#    && tar -xzf boost_1_73_0.tar.gz \
-#    && cd boost_1_73_0 \
-#    && ./bootstrap.sh \
-#    && ./b2 define=_GLIBCXX_USE_CXX11_ABI=1 threading=multi --prefix=/usr -j 64 cxxflags=-fPIC cflags=-fPIC install || true \
-#    && cd .. \
-#    && rm -rf boost_1_73_0.tar.gz \
-#    && rm -rf boost_1_73_0 \
-#    && cd /usr/include/boost
-
-# # add smdataparallel
-# # smdataparallel
-# ARG SMDATAPARALLEL_BINARY=https://smdataparallel.s3.amazonaws.com/binary/tensorflow/2.10.0/cu112/2022-09-08/smdistributed_dataparallel-1.4.1-cp39-cp39-linux_x86_64.whl
-
-# # Install SMD DP binary
-# RUN SMDATAPARALLEL_TF=1 ${PYTHON} -m pip install --no-cache-dir ${SMDATAPARALLEL_BINARY}
-
-# # Add NGC vars
-# ENV TF_AUTOTUNE_THRESHOLD=2
-
-# ENV LD_LIBRARY_PATH="/usr/local/lib/${PYTHON}/site-packages/smdistributed/dataparallel/lib:$LD_LIBRARY_PATH"
-
-# # remove tmp files
-# RUN rm -rf /tmp/*
-
-# RUN HOME_DIR=/root \
-#    && curl -o ${HOME_DIR}/oss_compliance.zip https://aws-dlinfra-utilities.s3.amazonaws.com/oss_compliance.zip \
-#    && unzip ${HOME_DIR}/oss_compliance.zip -d ${HOME_DIR}/ \
-#    && cp ${HOME_DIR}/oss_compliance/test/testOSSCompliance /usr/local/bin/testOSSCompliance \
-#    && chmod +x /usr/local/bin/testOSSCompliance \
-#    && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
-#    && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} ${PYTHON} \
-#    && rm -rf ${HOME_DIR}/oss_compliance*
+FROM common AS sagemaker
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+#SMDEBUGGER TESTING PLEASE REMOVE
+ARG TF_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/tensorflow/r2.11_aws/gpu/2022-12-22-20-41/tensorflow_gpu-2.11.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+
+# sagemaker-specific environment variable
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_tensorflow_container.training:main
+
+# https://github.com/tensorflow/models/issues/9267
+# tf-models does not respect existing installations of TF and always installs open source TF
+RUN ${PIP} install --no-cache-dir -U \
+    tf-models-official==2.11.0
+
+RUN ${PIP} install --no-cache-dir -U \
+    tensorflow-text==2.11.0
+
+RUN ${PIP} uninstall -y tensorflow tensorflow-gpu \
+  ; ${PIP} install --no-cache-dir -U \
+    ${TF_URL} \
+    tensorflow-io==0.27.0 \
+    tensorflow-datasets==4.7.0
+
+# Install Horovod, temporarily using CUDA stubs
+RUN ldconfig /usr/local/cuda-11.2/targets/x86_64-linux/lib/stubs \
+ && HOROVOD_GPU_ALLREDUCE=NCCL HOROVOD_WITH_TENSORFLOW=1 ${PIP} install --no-cache-dir horovod==0.26.1 \
+ && ldconfig
+
+RUN $PYTHON -m pip install --no-cache-dir -U \
+    "sagemaker>=2.93.0,<3" \
+    sagemaker-experiments==0.* \
+    "sagemaker-tensorflow-training>=20.3.1" \
+    "sagemaker-training>=4.2.8" \
+    "sparkmagic<1" \
+    "sagemaker-studio-sparkmagic-lib<1" \
+    "sagemaker-studio-analytics-extension==0.0.2" \
```

Comment:
This package currently seems to be at `0.0.14`. Can this be changed to `sagemaker-studio-analytics-extension<1.0`?